cmake/modules/FindLibDvd.cmake

```diff
@@ -202,18 +202,81 @@ if(NOT WIN32)
     set(LIBDVD_FOUND 1 CACHE BOOL "libdvd found" FORCE)
     endif()
 else()
-  # Dynamically loaded on Windows
-  find_path(LIBDVD_INCLUDE_DIR dvdcss/dvdcss.h)
-
+  # windows specific builds
+  find_program(PATCH patch)
   include(FindPackageHandleStandardArgs)
-  find_package_handle_standard_args(LibDvd REQUIRED_VARS LIBDVD_INCLUDE_DIR)
+  find_package_handle_standard_args(PATCH REQUIRED_VARS PATCH)
```

Comment:
this is not the correct place for searching a patch binary. Please move to windows depends